README.md

```diff
@@ -1,2 +1,62 @@
-# leaderboard
-Leaderboard app interacting with an API meant to practice async Javascript
+# Leaderboard
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+
+> Leaderboard app that interacts with an API for the purpose of practicing async Javascript
+
+![screenshot](./app_screenshot.png)
```

Comment:
- [ ] Let's add a screenshot of the app or remove the screenshot until we have a screen that we are proud of. Anything to keep your README professional :sweat_smile: